internal/agent/config.go

```diff
@@ -83,6 +84,14 @@ type Cloud struct {
 	ResourceID   string `hcl:"resource_id,optional"`
 }
 
+// ExporterConfig holds
```

Comment:
nit: comment unfinished 
```suggestion
// ExporterConfig holds configuration options to export metrics to a desired custom endpoint.
```